core/conversion/conversion.cpp

```diff
@@ -480,11 +480,24 @@ bool VerifyConverterSupportForBlock(const torch::jit::Block* b) {
     unsupported_msg << "Method requested cannot be compiled by TRTorch.\nUnsupported operators listed below:"
                     << std::endl;
     for (auto s : unsupported_ops) {
-      unsupported_msg << "  -  " << s << std::endl;
+      unsupported_msg << "  - " << s.second << std::endl;
     }
     unsupported_msg << "You can either implement converters for these ops in your application or request implementation"
                     << std::endl;
     unsupported_msg << "https://www.github.com/nvidia/TRTorch/issues" << std::endl;
+    unsupported_msg << std::endl << "In Module:" << std::endl;
+
+    for (const auto n : b->nodes()) {
+      auto schema = n->maybeSchema();
+      if (schema) {
+        for (const auto& x : unsupported_ops) {
+          if (x.first == schema->operator_name()) {
+            unsupported_msg << "  Unsupported operator: " << *schema << std::endl;
+            unsupported_msg << trtorch::core::util::GetPyTorchSourceCode(n) << std::endl;
+          }
+        }
+      }
+    }
     LOG_ERROR(unsupported_msg.str());
```

Comment:
The only suggestion I have is we print each instance of an unsupported op with LOG_ERROR to clearly delineate each case. 

Doing something like this:

```c++
    unsupported_msg << std::endl << "In Module:" << std::endl;

    LOG_ERROR(unsupported_msg.str());

    for (const auto n : b->nodes()) {
      auto schema = n->maybeSchema();
      if (schema) {
        for (const auto& x : unsupported_ops) {
          if (x.first == schema->operator_name()) {
            LOG_ERROR("Unsupported operator: " << *schema << std::endl << trtorch::core::util::GetPyTorchSourceCode(n) << std::endl);
          }
        }
      }
    }
    return false;
```
produces 
![image](https://user-images.githubusercontent.com/1790613/126675337-822ef35e-30f5-4998-bf81-1fd9532b560f.png)
which i think is more readable

Reply:
Resolve in your patch